nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/io/marshallers/json/AggregateJsonWriter.java

```diff
@@ -156,9 +157,11 @@ protected void writeEntityBody(Aggregate agg, JsonGenerator jg) throws IOExcepti
                                                      "label")
                                              .with(MAX_DEPTH_PARAM, "max")
                                              .open()) {
-
-                    writeBuckets("buckets", agg.getBuckets(), field, jg);
-                    writeBuckets("extendedBuckets", agg.getExtendedBuckets(), field, jg);
+                    // write buckets with privileges because we create property to leverage marshallers
```

Comment:
I'd say:
// write buckets with privilege because we create a property to leverage marshallers